styles/main.css

```diff
@@ -202,10 +203,9 @@ main{
     filter: grayscale(100%);
 }
 .row-span-2{
-    grid-row:span 2;
+    grid-row: span 2;
 }
 .col-span-2{
-    grid-row: span 2;

```

Comment:
Now that the 10th image has both the row-span-2 AND col-span-2 classes, there's no need for this grid-row declaration.  This makes the class more flexible.  We could now add this class to any element that needs to span 2 grid columns and it would Just Work.